src/views/DocumentationTopic.vue

```diff
@@ -112,13 +113,14 @@ export default {
       topicDataDefault: null,
       topicDataObjc: null,
       isSideNavOpen: false,
+      version: null,
       store: DocumentationTopicStore,
       BreakpointName,
     };
   },
   computed: {
-    objcOverrides: ({ topicData }) => {
-      const { variantOverrides = [] } = topicData || {};
+    objcOverrides: ({ versionedTopicData }) => {
```

Comment:
In case you haven't already, you may want to do some testing with a real Swift/Objective-C project to test that the versioning behavior works for that as well as for Swift-only codebases.